### ISSUE TITLE
OSDC-JODC Collaborated Meetup

### ISSUE THREAD

**Issue by Ankk98** [osdc/meetups#23]
## Title : 
OSDC-JODC Collaborated Meetup

## Abstract

Let us have a collaboration to strengthen community bonding.

## Expected Duration
 
2-3 hours

## Track

To be decided

## Talk Type
- [ ] Free Talk
- [ ] Talk + Hands-On
- [x] Interaction + Shraing experiences + Fun to play challenge 

## Pre-requisites

To be decided

## Resources

N.A.

## Slides

N.A.

**Comment by ryzokuken** [comment]
@sidntrivedi012 @code-monk08 @Yugandhartripathi @nightwarrior-xxx

**Comment by sidntrivedi012** [comment]
@Ankk98 Sure, it will be really great to have a collaborated meetup indeed.

As discussed, it will be great if we organize it after T1. Do you have any dates in mind when we can have it? OSDC has its meetup every Tuesday, so if we go by it we have 24th Sep and then 1st Oct. But, we can also organize it on any other day which may be convenient for everyone. Thanks.

**Comment by Ankk98** [comment]
@sidntrivedi012 I think dates can be deceided later, we should first decide what this meetup will exactly be like.

Some points to discuss on:

1. Should this meetup be only for active members or whole community

2. What fun to play hack-ctivity can we have?

3. I think this should not be a talk by 1-2 people, but maybe experience sharing by 10-15 people in 2-5 mins each

Eg. We can have all gsocers to share their experiences
Eg. We can have people of different feilds to talk about their field

Basically i want everyone to be actively involved and we all can get to know each others personally and professionally.
I also want to make this a memorable experience.
Ideally if we want everyone to be actively involved, we should have limited number of people, like maybe 30. So, we can keep it closed to active/core/invited members only.

Everyone share your ideas on this.

**Comment by sidntrivedi012** [comment]
> @sidntrivedi012 I think dates can be deceided later, we should first decide what this meetup will exactly be like.

Sure, we can decide them later.
 
> Some points to discuss on:
> 
> 1. Should this meetup be only for active members or whole community

I think we should keep it open to anyone who wants to attend.

> 2. What fun to play hack-ctivity can we have?

We can organize a small wargame / a mini hackathon. We can assemble more ideas for sure.
 
> 3. I think this should not be a talk by 1-2 people, but maybe experience sharing by 10-15 people in 2-5 mins each
>
> Eg. We can have all gsocers to share their experiences
> Eg. We can have people of different fields to talk about their field

Sure. We can have two dedicated talks and then a sort of Free session/ where everyone can share their experience in open source. Also, we can have a Panel Discussion and also a kinda AMA for anyone to get their queries resolved.

**Comment by Ankk98** [comment]
Yeah, so we can choose any of these models:

1. Limited member/invite-only one with the goal that everyone gets to speak and heard. Everyone gets to know everyone else present there both formally and personally. We can even randomly team up people for the "hack-games". We can even contribute money for something like pizzas. Can be added with some debates/discussions or anything. Even location outside college will be possible.
But this will be an "Experimental" meetup, so do understand that.

2. Open to all, with limited speakers with some "lightning talks", panel discussions.

3. Old school, monotonous speaker-listener model with 1-2 speaker.

4. Other

Everyone share your views.
Do share your wild ideas. :)

**Comment by code-monk08** [comment]
> Everyone gets to know everyone else present there both formally and personally.  We can even randomly team up people for the "hack-games".
This is some thing which will be useful in the long run, pairing up the attendees in random groups and organizing hack-games would be fruitful, the best possible event could be a war-game. 

> We can even contribute money for something like pizzas.
will it be necessary to lure them in? interested people will come anyway.  

> Can be added with some debates/discussions or anything. 
+1

> Even location outside college will be possible.
I would suggest we start with college, since this is an experimental meetup.

> Open to all, with limited speakers with some "lightning talks", panel discussions.
We can go with lightning talks, panel discussions may stretch for a longer duration than expected. 
 
> Old school, monotonous speaker-listener model with 1-2 speaker.
Any way we can make it interactive?

**Comment by eldraco19** [comment]
We can set a Pokemon showdown server, 
https://github.com/Zarel/Pokemon-Showdown

**Comment by Ankk98** [comment]
> will it be necessary to lure them in? interested people will come anyway.

Not to lure, but can act as a small break. Although not necessary.
Anyway everyone present will be contributing.

**Comment by Ankk98** [comment]
@eldraco19 Tum logo ka long pending idea of informal meet bhi ban sakta hai yeh. :)

> We can setup a Pokemon showdown server

I think we should prefer something more technical but yeah anything like this could also work if everyone finds it good.

**Comment by rexagod** [comment]
Looping in @koderjoker @VaRoDeK

**Comment by Ankk98** [comment]
@vaibhavk @amanharitsh123 @im-gozmit @octamois @Ankurk99 @yellowwoods12

**Comment by koderjoker** [comment]
@sidntrivedi012 Actually earlier this week we had already officially registered our Hactoberfest event with DigitalOcean for October 1, with the confirmed venue as JIIT 128.

Along with that, if we have a collab for our first Hacktoberfest event at JIIT 62 I fear that we'd be leaving out day scholars and pg students from our college. Same applies for if we hold the collab at 128.

How about extending our Hacktoberfest events to a 2nd or 3rd day, and hold our collab that day? 

Additionally, so that we could quickly have a discussion, clarify all the details and reach a conclusion, would it be possible to have a zoom meeting together after T1 at a time when everyone's free?

**Comment by sidntrivedi012** [comment]
>How about extending our Hacktoberfest events to a 2nd or 3rd day, and hold our collab that day?

@koderjoker Sure. We can have the collab later. Maybe around 3rd/4th October.

Yes, we can have a community call after T1 (maybe on 19?) to sort all the details. I guess everyone would be free then.

**Comment by vaibhavk** [comment]
Is this issue resolved? Any further updates?

**Comment by daemon1024** [comment]
We have been having collaborative meetups for the past one year cause it was all online :`) 
We have streamlined our meetups and now it happens on Saturday, 10PM every week, it would be great to have yall there. 

Closing this issue for now :D Thank you for all the efforts for this initiative.

**Comment by rexagod** [comment]
Venue: https://discord.gg/DH3fGkz2jr
